dask_image/utils/_dispatcher.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import scipy.ndimage.filters
+from dask.utils import Dispatch
+
+
+class Dispatcher(Dispatch):
+    """Simple single dispatch for different dask array types."""
+
+    def __call__(self, arg, *args, **kwargs):
+        """
+        Call the corresponding method based on type of dask array.
+        """
+        meth = self.dispatch(type(arg._meta))
+        return meth(arg, *args, **kwargs)
+
+
+convolve_dispatch = Dispatcher(name="convolve_dispatch")
+
+
+@convolve_dispatch.register(np.ndarray)
+def numpy_convolve(*args, **kwargs):
+    return scipy.ndimage.filters.convolve
```

Comment:
```suggestion
    return scipy.ndimage.filters.convolve(*args, **kwargs)
```

Reply:
Is this really what we want here? I had thought the bare function itself was what we wanted returned, and that's what gets put in as the first argument to `map_overlap`. I didn't try it this way, though.

Reply:
That's typically how it is used like [with `sizeof`]( https://github.com/dask/dask/blob/72de42cdde88ba01b4d762147e4105892a6535a2/dask/sizeof.py#L17-L22 ).

As `Dispatch` instances (like `convolve_dispatch`) should behave like a function, I think this should work here. Though please let me know if that is not the case.

Reply:
I think that's because `sizeof` isn't using `map_overlap` or similar, so you need to pass the arguments into the function for it to work. Whereas here we have `image.map_overlap(func, *args, *kwargs)` and I am trying to replace func with the dispatch object (which should return `func` as a bare function object that is ingested into `map_overlap`).

I might just be confused about what you're trying to say here though